components/shared/loanMaterialModal/LoanMaterialModal.tsx

```diff
@@ -96,6 +103,21 @@ const LoanMaterialModal = ({
     (isCostFree && canUserLoanMoreCostFreeMaterials(dataLoans)) ||
     canUserLoanMoreMaterials(dataLoans, dataLibraryProfile, manifestation)
 
+  const errors: string[] = []
+  if (!isLoanPossible) {
+    errors.push(
+      `Du kan desværre ikke låne flere titler af typen "${getManifestationMaterialTypeSpecific(manifestation)}" i denne måned.`
+    )
+  }
+  if (isErrorLoans || isErrorLibraryProfile) {
+    errors.push(
+      "Vi kunne desværre ikke bekræfte, om du kan låne materialet. Prøv venligst igen senere."
+    )
+  }
+  if (publizonError) {
+    errors.push(publizonErrorMessageMap[publizonError.code] || publizonError.message)
+  }
+
```

Comment:
I think it could make sense to move the errors logic in separate helper function. The `LoanMaterialModal` is way too big in my taste. We cannot write unit tests for the various parts and I honestly start to get worried if I get a task where I need to work on this part of the application.